apps/webapp/src/state/staking.ts

```diff
@@ -219,6 +220,43 @@ export const createStakingSlice = (): SliceCreator<StakingSlice> => (set, get) =
       }
     }
   },
+  undelegate: async () => {
+    const toast = new TransactionToast('undelegate');
+    toast.onStart();
+
+    try {
+      const transactionPlan = await plan(assembleUndelegateRequest(get().staking));
+
+      // Reset form _after_ building the transaction form, since it depends on
```

Comment:
Is it after the build or after planning?

Reply:
Whoops, I had a typo — the second "form" wasn't supposed to be there. I also reword the comment — it now reads:
```ts
      // Reset form _after_ assembling the transaction planner request, since it
      // depends on the state.
```